<launch>
    <node name="task_creator" pkg="task_manager" type="task_creator.py" output="screen"/>
</launch>